Formula/rakudo-star.rb

```diff
@@ -1,13 +1,13 @@
 class RakudoStar < Formula
   desc "Rakudo compiler and commonly used packages"
   homepage "https://rakudo.org/"
-  url "https://rakudo.org/dl/star/rakudo-star-2021.04.tar.gz"
-  sha256 "66a5c9d7375f8b83413974113e1024f2e8317d8a6f505e6de0e54d5683c081e7"
+  url "https://github.com/rakudo/star/releases/download/2022.02/rakudo-star-2022.02.tar.gz"
+  sha256 "49a2f9d440ffd443e59bf52b414220e4186b28c27a1984331d207d4c0e9b0968"
   license "Artistic-2.0"
 
   livecheck do
-    url "https://rakudo.org/dl/star/"
-    regex(/".*?rakudo-star[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    url :url
+    strategy :git
   end
```

Comment:
we probably can get rid of this if we are using github artifact.

Reply:
I have pushed an update with the livecheck removed. Thanks for the review.